### DISCUSSION TITLE
tbselenium

### DISCUSSION THREAD

**Discussion by mozbox264** [webfp/tor-browser-selenium#194] [Q&A] [answered]
Hello. I'm sorry. Installed tbselenium . I did everything according to the instructions, but I get an error:

![Снимок экрана от 2023-08-07 14-56-11](https://github.com/webfp/tor-browser-selenium/assets/141633146/8373b903-fccf-4dfd-ba76-01be21e5756c)

What did I do wrong? Help please

**Comment by gunesacar** [comment, marked as answer]
Could you please check if the troubleshooting instructions help? https://github.com/webfp/tor-browser-selenium#-troubleshooting
My guess would be you have some unmet dependencies  such as `selenium` or `geckodriver`

**Reply by mozbox264** [reply]
I installed geckodriver 0.31.0; 0.32.0. It doesn't work. Selenium is installed. 7 days ago it worked without problems. Reinstalled the linux system, tor. Now I can't configure it. geckodriver --version 0.31.0 in the system. I've been sitting for 2 days, I don't know what to do.

**Reply by gunesacar** [reply]
Can you share your selenium version?

**Reply by mozbox264** [reply]
4.11.2

**Comment by gunesacar** [comment]
Looks like selenium 4.11(.?) introduced a breaking change. Can you please uninstall selenium, and install selenium 4.10.0?
Something like the following should do:
`pip install selenium==4.10.0`

**Reply by mozbox264** [reply]
you're right. everything works on selenium 4.10.0. Thanks 👍👍👍
